app/views/pages/remiss.html

```diff
@@ -1,19 +1,21 @@
-<h2 style="text-align: center;">Remiss</h2>
-
-<hr />
-<p>Vi &auml;r ett n&auml;tverk av yrkeskunniga f&ouml;retagare som alla &auml;r seri&ouml;sa och ser till hundens b&auml;sta. Kom ih&aring;g att vi alla gynnas av bransch&ouml;verskridande samarbeten.</p>
-
-<p>Remittera g&auml;rna till varandra genom att lotsa kunder med behov av andra tj&auml;nster vidare till en kollega inom SHF. Se till att h&aring;lla en n&auml;ra kontakt och delge varandra journalanteckningar, iakttagelser, rekommenderade behandlingar och liknande s&aring; att kunden f&aring;r maximal hj&auml;lp tack vare ert samspel. Ett snabbt telefonsamtal mellan en behandlande hundfysioterapeut och hundpsykolog kan g&ouml;ra stor skillnad f&ouml;r det enskilda fallet.</p>
-
-<p><a href="/hundforetag">hitta mailadress</a></p>
-
-<h3>Information att delge vid remittering:</h3>
-
-<ul>
-	<li>Hund&auml;garens namn:</li>
-	<li>Hund&auml;garens kontaktdetaljer:</li>
-	<li>Hundnamn, ras/raser, &aring;lder, k&ouml;n:</li>
-	<li>Orsak till remiss:</li>
-</ul>
-
-<p>N&auml;r du tar dig an en remiss &auml;r det viktigt att &aring;terkoppla till remitterande kollega.</p>
+<p>Vi &auml;r ett n&auml;tverk av yrkeskunniga f&ouml;retagare som alla &auml;r seri&ouml;sa och ser till hundens b&auml;sta. Kom ih&aring;g att vi alla gynnas av bransch&ouml;verskridande samarbeten.</p>
+
+<p>Remittera g&auml;rna till varandra genom att lotsa kunder med behov av andra tj&auml;nster vidare till en kollega inom SHF. Se till att h&aring;lla en n&auml;ra kontakt och delge varandra journalanteckningar, iakttagelser, rekommenderade behandlingar och liknande s&aring; att kunden f&aring;r maximal hj&auml;lp tack vare ert samspel. Ett snabbt telefonsamtal mellan en behandlande hundfysioterapeut och hundpsykolog kan g&ouml;ra stor skillnad f&ouml;r det enskilda fallet.</p>
+
+<p><a href="/hundforetag">hitta mailadress</a></p>
+
+<h3>Information att delge vid remittering:</h3>
+
+<ul>

```

Comment:
Minor _opinion_ comment:

I think this should be a numbered list (`%ol`).  The bullets make it less clear that each item is really like a _label_ and folks should fill out info after them.
I think this is clearer:

1. Hund&auml;garens namn:
2. Hund&auml;garens kontaktdetaljer:
3. Hundnamn, ras/raser, &aring;lder, k&ouml;n:
4. Orsak till remiss:

(but with more blank lines/space after " Hund&auml;garens kontaktdetaljer:" as needed.)

Most forms have numbered items to fill out, not bullets.